Cargo.toml

```diff
@@ -266,6 +266,11 @@ name = "ring"
 libc = "0.2.20"
 untrusted = "0.3.2"
 
+[build-dependencies]
+gcc = "0.3"
```

Comment:
We don't use `gcc = { version = "0.3", features = ["parallel"] }` because we do the parallelism ourselves, right? If that is the case, let's add a comment to that effect here.